### PR TITLE
Fix Git History

### LINKED ISSUE #1024: No Git history for selected file
## Description

Unable to view Git history for a selected file. Implemented in https://github.com/jupyterlab/jupyterlab-git/commit/02c6d9aa8cf3344b37397e303c8d345efab12aff for  https://github.com/jupyterlab/jupyterlab-git/issues/864

## Reproduce

1. Clone an active repo 
2. Right click on a file (containing history) and navigate to `Git` -> `History`
3. `No history found` shows on file

## Expected behavior

Expect to view a number of Git history commits based on the value set for `historyCount` under advanced settings (kept default of 25) 

## Context

```
# cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 10 (buster)"

# pip list | grep jupyterlab-git
jupyterlab-git         0.32.2

# jupyter labextension list
JupyterLab v3.0.17
  @jupyterlab/git v0.32.2 enabled OK (python, jupyterlab-git)

# jupyter server extension list
  jupyterlab_git enabled
  - Validating jupyterlab_git...
     jupyterlab_git 0.32.2 OK

# git --version
git version 2.20.1
```

### PR DESCRIPTION
**Pull request by fcollonval**
No Git history for selected file - Fixes #1024
Don't display context menu items if the current folder is not a Git repository.

### CODE REVIEW

**Comment by fcollonval** [comment]
@navn-r would you like to review this one?

File: tests/model.spec.tsx [hunk 1]
```diff
@@ -269,32 +269,57 @@ describe('IGitExtension', () => {
     });
   });
 
+  describe('#getFile', () => {
+    it.each([
```
Comment (navn-r) [inline tests/model.spec.tsx:273]: Oh this is pretty cool, didn't know Jest had something like this :+1:

**Review by navn-r — APPROVED** [review]
Thanks @fcollonval

**Comment by fcollonval** [comment]
Thanks for the review @navn-r

**Comment by fcollonval** [comment]
@meeseeksdev please backport to 0.32.x

**Pull request merged**